client/landing/jetpack-cloud/sections/auth/connect/index.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+import JetpackLogo from 'components/jetpack-logo';
+import Main from 'components/main';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	authUrl: string;
+}
+
+const Connect: FunctionComponent< Props > = ( { authUrl } ) => {
+	const translate = useTranslate();
+
+	return (
+		<Main className="connect">
+			<div className="connect__content">
+				<JetpackLogo full monochrome={ false } size={ 72 } />
+				<p>
+					{ translate(
+						'Welcome to Jetpack Cloud. Authorize with your WordPress.com credentials to get started.'
+					) }
+				</p>
+				<Button primary href={ authUrl }>
+					{ translate( 'Authorize Jetpack Cloud' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( 'Authorize Jetpack' )` *ES Score: 6*